app/models/recommendation.js

```diff
@@ -0,0 +1,45 @@
+import DS from 'ember-data';
+import { attr, belongsTo } from '@ember-decorators/data';
+
+const { Model } = DS;
+
+export default class RecommendationModel extends Model {
+// DB table: dcp_communityboarddisposition
+
+  @belongsTo('user') user;
```

Comment:
I think it belongs to action, not user. One user can have multiple recommendations, one recommendation per action.

Reply:
Okay so the relationships are: 1) One Recommendation to One Action **_OR_** 2) One Recommendation to Many Actions